frontend/ui/widget/opdsbrowser.lua

```diff
@@ -461,6 +464,15 @@ function OPDSBrowser:appendCatalog(item_table_url)
     return true
 end
 
+function OPDSBrowser:openBook(file_downloaded)
```

Comment:
I think I'd stick the word path somewhere in the variable name; otherwise you'd have to read the code calling this to know it's not an object but just a simple string.